3기/LeeGaYeon/week1/page1/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="item"><p>Header</p></header>
+    <div class="item container">
+      <aside><p>Sidebar</p></aside>
+      <main id="posting-container">
+        <div class="posting"><p>Posting1</p></div>
+        <div class="posting"><p>Posting2</p></div>
+        <div class="posting"><p>Posting3</p></div>
+        <div class="posting"><p>Posting4</p></div>
+      </main>
+    </div>
+    <footer class="item"><p>Footer</p></footer>
```

Comment:
`<header>`, `<aside>`, `<footer>` 태그의 사용은 좋은데, Heading 태그가 없는게 아쉬워요!
각 요소에 적절한 레벨의 헤딩태그를 넣어주면 검색엔진 최적화에 더 많은 도움이 돼요!

```suggestion
    <header class="item"><h1>Header</h1></header>
    <div class="item container">
      <aside><h2>Sidebar</h2></aside>
      <main id="posting-container">
        <div class="posting"><p>Posting1</p></div>
        <div class="posting"><p>Posting2</p></div>
        <div class="posting"><p>Posting3</p></div>
        <div class="posting"><p>Posting4</p></div>
      </main>
    </div>
    <footer class="item"><h2>Footer</h2></footer>
```